test/utils.js

```diff
@@ -752,25 +750,25 @@ describe('utils.htmlToText()', () => {
         /* eslint-disable */
         checkHtmlToText(
             html1,
-            `Let's start with a simple text. 
-The ships hung in the sky, much the way that bricks don't. 
+            `Let's start with a simple text.
```

Comment:
@mtrunkat There's apparently something in your IDE / Lint config that keeps removing whitespace in this string, which breaks the test.

Reply:
I am trying to reconfigure it but it still removes all the spaces so I am not able to revert that :)